lib/statsd.js

```diff
@@ -107,6 +107,34 @@ Client.prototype.timing = function (stat, time, sampleRate, tags, callback) {
   this.sendAll(stat, time, 'ms', sampleRate, tags, callback);
 };
 
+/**
+ * Function that records the time to run a given function
```

Comment:
Perhaps something like: `Represents the timing stat by recording the duration a function takes to run (in milliseconds)`

This way the description indicates what metrics will be output, similar to other descriptions